near-plugins/tests/contracts/access_controllable/src/lib.rs

```diff
@@ -4,11 +4,15 @@ use near_sdk::serde::{Deserialize, Serialize};
 use near_sdk::{env, log, near_bindgen, AccountId};
 use std::collections::HashMap;
 
+/// Roles are represented by enum variants.
 #[derive(AccessControlRole, Deserialize, Serialize, Copy, Clone)]
 #[serde(crate = "near_sdk::serde")]
 pub enum Role {
+    /// Represents LevelA.
     LevelA,
+    /// Represents LevelB.
     LevelB,
+    /// Represents LevelC.
```

Comment:
Adding not-so-meaningful comments to show it's now working and to avoid regressions.

Reply:
Just a note, we can add these comments in an enum in some of the examples to test the regression.